README.md

```diff
@@ -12,13 +12,13 @@ The contents of this code of conduct apply to our interactions in various areas
 
 ## Values
 
-We want our team to be a fun, productive, and safe space for all members. In addition to core values of respect and honesty, there are several ways in which our product team distinguishes itself.
+We want our team to be fun, productive, and safe for all members. In addition to the Social Tables [core values](https://www.socialtables.com/company/about/#values), there are several ways in which our Engineering team distinguishes itself.
```

Comment:
Totally. It's entirely possible to MVC this: adopt it for the Engineering team (and fellow travelers), and abide by it for several months/work out the bugs. By the spring we'll have experience with it and could, with confidence, introduce it to the company at-large if we chose.